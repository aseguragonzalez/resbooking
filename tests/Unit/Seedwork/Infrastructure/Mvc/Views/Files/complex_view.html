<section>
    <header>
        <h1>{{model->title}}</h1>
    </header>
    <article>
        <h2>Customer Details</h2>
        <ul>
            <li>{{model->customer->name}}</li>
            <li>{{model->customer->taxId}}</li>
            <li>{{model->customer->phone}}</li>
        </ul>
        <address>
            <h3>Address</h3>
            <p>{{model->customer->address->street}}, {{model->customer->address->number}}</p>
            <p>{{model->customer->address->city}}</p>
            <p>{{model->customer->address->state}}</p>
        </address>
        <h3>Transactions</h3>
        <ul>
            {{#for transaction in model->customer->transactions:}}
            <li>({{transaction->createdAt}}) {{transaction->amount}}â‚¬ | {{transaction->status}}</li>
            {{#endfor model->customer->transactions:}}
        </ul>
    </article>
    <footer>
        <p>
            Total amount: {{model->totalAmount}}<br>
            Updated at: {{model->updatedAt}}
        </p>
    </footer>
</section>
