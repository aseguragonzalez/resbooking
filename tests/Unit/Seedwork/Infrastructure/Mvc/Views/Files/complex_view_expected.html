<section>
    <header>
        <h1>Transactions</view>
    </header>
    <article>
        <h2>Customer Details</h2>
        <ul>
            <li>Peter Parker</li>
            <li>123.456.789-00</li>
            <li>(11) 98765-4321</li>
        </ul>
        <address>
            <h3>Address</h3>
            <p>Elm Street, 123</p>
            <p>Springwood</p>
            <p>Ohio</p>
        </address>
        <h3>Transactions</h3>
        <ul>
            <li>(+2025-01-02T12:01:02+00:00) 123.45€ | approved</li>
            <li>(+2025-01-03T12:01:02+00:00) 67.89€ | declined</li>
            <li>(+2025-01-04T12:01:02+00:00) 67.89€ | declined</li>
            <li>(+2025-01-05T12:01:02+00:00) 167.89€ | approved</li>
        </ul>
    </article>
    <footer>
        <p>
            Total amount: 1234.56<br>
            Updated at: +2025-01-01T12:01:02+00:00
        </p>
    </footer>
</section>
